.travis.yml

```diff
@@ -85,7 +85,7 @@ matrix:
             - libc6-dbg:i386
     # S390x build (big endian system)
     - compiler: gcc
-      env: HOST=s390x-unknown-linux-gnu ECDH=yes RECOVERY=yes EXPERIMENTAL=yes CTIMETEST=
+      env: HOST=s390x-unknown-linux-gnu ECDH=yes RECOVERY=yes CTIMETEST=
```

Comment:
Can you leave the experimental but add `SCHNORRSIG=yes` in both of these? (I can open a separate PR for that if you prefer)

Reply:
good catch, done